_sass/module/_candidate-summary.scss

```diff
@@ -13,27 +13,46 @@
   @include grid-column(10);
   margin-bottom: $spacing-base * 2;
   margin-top: $spacing-base * 2;
+
+  a {
+    text-decoration: none;
+  }
 }
 
 .candidate-summary__photo {
   width: 100%;
 }
 
+.candidate-summary__name {
+  font-size: $h3-font-size;
+  line-height: normal;
+  background-image: url('./assets/images/icon_more_info.png');
```

Comment:
What's the background image for?

Reply:
Thanks for this. This should have been applied to a different tag. My next commit will reflect my old thinking - but better.